R/esqlabs-env.R

```diff
@@ -31,8 +35,8 @@ esqlabsEnv$heightPerPlotMapping <- 8
 #' getEsqlabsRSetting("packageVersion")
```

Comment:
This example doesn't work. 

```r
> getEsqlabsRSetting("packageVersion")
Error in getEsqlabsRSetting("packageVersion") : 
  object 'ospsuiteEnv' not found
```

Is this a copy-paste error? Shouldn't `ospsuiteEnv` be `esqlabsEnv` in the code below?